frontend/appflowy_flutter/lib/user/application/sign_in_bloc.dart

```diff
@@ -253,6 +258,14 @@ class SignInEvent with _$SignInEvent {
   const factory SignInEvent.deepLinkStateChange(DeepLinkResult result) =
       DeepLinkStateChange;
   const factory SignInEvent.cancel() = _Cancel;
+  const factory SignInEvent.switchLoginType(LoginType type) = _SwitchLoginType;
```

Comment:
```suggestion
  const factory SignInEvent.toggleLoginType() = _ToggleLoginType;
```
_personal preference_